tests/integration/test_dynamodb.py

```diff
@@ -53,3 +55,80 @@ def test_large_data_download(self):
+        table = dynamodb.Table(TEST_DDB_TABLE_NAME_3)
+
+        # Insert some items to the table
+        items = {
+            'id1': {PARTITION_KEY: 'id1', 'data': 'IT IS'},
+            'id2': {PARTITION_KEY: 'id2', 'data': 'TIME'},
+            'id3': {PARTITION_KEY: 'id3', 'data': 'TO LIVE!'}
+        }
+        for k, item in items.items():
+            table.put_item(Item=item)
+
+        # Describe TTL when still unset.
+        response = testutil.send_describe_dynamodb_ttl_request(TEST_DDB_TABLE_NAME_3)
+        assert response.status_code == 200
+        assert json.loads(response._content)['TimeToLiveDescription']['TimeToLiveStatus'] == 'DISABLED'
+
+        # Enable TTL for given table
+        response = testutil.send_update_dynamodb_ttl_request(TEST_DDB_TABLE_NAME_3, True)
+        assert response.status_code == 200
+        assert json.loads(response._content)['TimeToLiveSpecification']['Enabled'] is True
+
+        # Describe TTL status after being enabled.
+        response = testutil.send_describe_dynamodb_ttl_request(TEST_DDB_TABLE_NAME_3)
+        assert response.status_code == 200
+        assert json.loads(response._content)['TimeToLiveDescription']['TimeToLiveStatus'] == 'ENABLED'
+
+        # Disable TTL for given table
+        response = testutil.send_update_dynamodb_ttl_request(TEST_DDB_TABLE_NAME_3, False)
+        assert response.status_code == 200
+        assert json.loads(response._content)['TimeToLiveSpecification']['Enabled'] is False
+
+        # Describe TTL status after being disabled.
+        response = testutil.send_describe_dynamodb_ttl_request(TEST_DDB_TABLE_NAME_3)
+        assert response.status_code == 200
+        assert json.loads(response._content)['TimeToLiveDescription']['TimeToLiveStatus'] == 'DISABLED'
+
+        # Enable TTL for given table again
+        response = testutil.send_update_dynamodb_ttl_request(TEST_DDB_TABLE_NAME_3, True)
+        assert response.status_code == 200
+        assert json.loads(response._content)['TimeToLiveSpecification']['Enabled'] is True
+
+        # Describe TTL status after being enabled again.
+        response = testutil.send_describe_dynamodb_ttl_request(TEST_DDB_TABLE_NAME_3)
+        assert response.status_code == 200
+        assert json.loads(response._content)['TimeToLiveDescription']['TimeToLiveStatus'] == 'ENABLED'
+
+        # Clean up table
+        dynamodb_client.delete_table(TableName=TEST_DDB_TABLE_NAME_3)
+
+    def test_tag_resource(self):
+        response = testutil.send_dynamodb_request('', action='TagResource', request_body=json.dumps({
+            'ResourceArn': testutil.get_sample_arn('dynamodb', 'table'),
+            'Tags': [{'tagkey1': 'tagvalue1'}, {'tagkey2': 'tagvalue2'}, {'tagkey3': 'tagvalue3'}]
+        }))
+        assert response.status_code == 200
+        assert response._content == ''  # Empty string if tagging succeeded (mocked for now)
```

Comment:
Looks like this assertion is failing in the CI build, possibly because `response._content` is `None` instead of `''`. Maybe change the assertion to:
```
assert not response._content
```
(which covers both cases).